modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1592,6 +1592,10 @@
 			<category name="Upgrade" />
 			<description name="Replaces the references of the method `HtmlUtil.extractText(` with the method `extractText(` of `HtmlParser` class" />
 		</check>
+		<check name="UpgradeJavaGetImagePreviewURLMethodCheck">
+			<category name="Upgrade" />
+			<description name="Replaces the references of the method 'DLUtil.getImagePreviewURL(' with the method 'getImagePreviewURL(' of 'DLURLHelper' class" />
```

Comment:
@TamyrisTorres, no need to include the `(` in `DLUtil.getImagePreviewURL(` and `getImagePreviewURL(`, after this is just documentation, right? :slightly_smiling_face:

Reply:
The change is done! @natocesarrego